network.go

```diff
@@ -1209,25 +1203,10 @@ func createPhysicalEndpoint(ifaceName string) (*PhysicalEndpoint, error) {
 	vendorDeviceID := fmt.Sprintf("%s %s", vendorID, deviceID)
 	vendorDeviceID = strings.TrimSpace(vendorDeviceID)
 
-	// Get mac address
-	netHandle, err := netlink.NewHandle()
-	if err != nil {
-		return nil, err
-	}
-	defer netHandle.Delete()
-
-	link, err := netHandle.LinkByName(ifaceName)
-	if err != nil {
-		return nil, err
-	}
-
-	mac := link.Attrs().HardwareAddr.String()
-	MTU := link.Attrs().MTU
-
 	physicalEndpoint := &PhysicalEndpoint{
-		IfaceName:      ifaceName,
-		HardAddr:       mac,
-		MTU:            MTU,
+		IfaceName:      netInfo.Iface.Name,
+		HardAddr:       netInfo.Iface.HardwareAddr.String(),
+		MTU:            netInfo.Iface.MTU,
```

Comment:
We should drop the MTU field from the Physical Endpoint now, since we can access it from endpoint.Properties().Iface.MTU now as we don't rely on the second scan for the MTU value.

Reply:
Yes I can probably do that :)

Reply:
Done